personal portfolio/portfolio.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="portfolio.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tanya Piña</title>
+</head>
+<body>
+  <header id="tanya-intro">
+    <a href="about.html">About Page</a>
+    <img src="https://miro.medium.com/max/640/1*ZzUGWgV_iSVhWHHtQVPHSw.webp" alt="Headshot of Tanya, a Mexican-American person wearing a smile and a burgundy shirt" width="400" height ="400">
+    <h1>Tanya Piña</h1>
+    <h2>Software Engineer in Training</h2>
+  </header>
+  <section id="tanya-about">
+    <p>
+      Hello! My name is Tanya, I use they/them pronouns and am a current Software Engineer in Training at Techtonica. 
+      I am passionate about integrating my soft and technical skills in order to create inclusive spaces by and for underrepresented communities. 
+      I love to read comics in my spare time and hope to make a zine website soon!
+    </p>
+  </section>
+  <section id="tanya-links">
+    <p>Find me here:</p>
+    <p id="iconlinks">
+      <a href="mailto:jtanyapina15@gmail.com" class="links"target="_blank">
+        <ion-icon name="mail-outline" id="mail"></ion-icon> 
+      </a>
+      <a href="https://github.com/TanyaPina" class="links"target="_blank">
+        <ion-icon name="logo-github" id="github"></ion-icon> 
+      </a>
+      <a href="https://www.linkedin.com/in/tanyapina/" class="links"target="_blank">
+        <ion-icon name="logo-linkedin" id="linkedin"></ion-icon> 
+      </a>
+      
+    </p>
+  </section>
+  <section id="contact-form">
+    <p>Or get in touch!</p>
+    <form id="contact-me-here">
+      <label for="name">Name:</label><br>
+      <input type="text" id="name" name="name" required><br>
+      <label for="email">Email:</label><br>
+      <input type="text" id="email" name="email" required><br>
+      <label for="phonenumber">Phone Number:</label><br>
+      <input type="text" id="phonenumber" name="phonenumber"><br>
+      <label for="address">Address:</label><br>
+      <input type="text" id="address" name="address"><br><br>
+      <button type="submit" id="submit">Submit</button>
+    </form>
+    <p id="welcome"></p>
```

Comment:
Is this leftover from a previous version? It does not appear to be displaying anything at the moment.

Reply:
Oh, I see. It's used by the Javascript file when the form is submitted. Nevermind.